packages/js-sdk/src/sandbox/sandboxApi.ts

```diff
@@ -99,15 +99,15 @@ export class SandboxApi {
     const config = new ConnectionConfig(opts)
     const client = new ApiClient(config)
 
-    let query = undefined
-    if (opts?.filters) {
-      const encodedPairs: Record<string, string> = Object.fromEntries(Object.entries(opts.filters).map(([key, value]) => [encodeURIComponent(key),encodeURIComponent(value)]))
-      query = new URLSearchParams(encodedPairs).toString()
+    let metadata = undefined
+    if (opts?.metadata) {
+      const encodedPairs: Record<string, string> = Object.fromEntries(Object.entries(opts.metadata).map(([key, value]) => [encodeURIComponent(key),encodeURIComponent(value)]))
+      metadata = new URLSearchParams(encodedPairs).toString()
     }
 
     const res = await client.api.GET('/sandboxes', {
         params: {
-          query: {query},
+          query: {metadata},
```

Comment:
query should be metadata

Reply:
that's query as a part of an url (the part after `?`)

Reply:
ahh, yes